Review.java

```diff
@@ -215,4 +215,30 @@ else if(totalSentiment(fileName) >= 30)
          return 0;
       }
    }
+  
+  public static String fakeReview(String fileName)
+	{
+		String text = textToString(fileName);
+  
+		for(int i = 0; i < text.length(); i++)
+		{
+			if(text.charAt(i) == '*')
+			{
+				int j = i;
+				while(true)
+				{
+					i++;
+					if(text.charAt(i) == ' ') 
+               {
+                  break;
+               }
+				}
+            int k = i;
+		
+				text = text.substring(0, j) + randomAdjective() + text.substring(k);
+			}
+		}
+		return text;
+	}
+  

```

Comment:
Looks correct, probably add some pseudo code to give a better idea of what your code is doing and the steps you took!